### DISCUSSION TITLE
Support for polars?

### DISCUSSION THREAD

**Discussion by kszlim** [hgrecco/pint#1833] [Ideas]
I was wondering if you see polars support ever coming? [Polars](https://github.com/pola-rs/polars/) is an up and coming DataFrame library written in rust with very nice Python bindings. Imo it's going to rival pandas in popularity in the future. Would be great if pint could integrate (though it might be tricky due to laziness in Polar's design).

**Comment by hgrecco** [comment]
It would be very nice! Take a look at pint pandas.

**Comment by kszlim** [comment]
Ah, I'm not sure I'll have time to implement it, do you know if there are any plans to implement `pint-polars`?
